### ISSUE TITLE
Nu Html Checker: Improve Html Quality

### ISSUE THREAD

**Issue by throwaway-d** [retypeapp/retype#583]
https://codeberg.org/rimgo/docs/issues/3
I've put Rimgo website through a website checker called Nu Html Checker for some clean HTML suggestions to improve Rimgo website. I contacted the owner of Rimgo website about those suggestions. But the owner told me that all HTML is written by Retype, the owner didn't have any control over generated HTML.

> Retype generates all the HTML, I can't do anything about this.

So I think I should report the suggestions to Retype instead, right?

Here are the HTML suggestions: https://validator.w3.org/nu/?showsource=yes&showoutline=yes&showimagereport=yes&doc=https%3A%2F%2Frimgo.codeberg.page%2Fdocs%2F

**Comment by geoffreymcgill** [comment]
Thanks for the report. 

The `type="text/javascript"` attribute has been removed from `<script>` tags in the upcoming Retype v3.1.0 release. 

Looks like we can easily remove the closing `/` forward-slash on the `<meta>` and `<input>` tags. I guess we're just more accustomed to writing XHTML instead of HTML5. 

Removing the `v-cloak` attribute is going to cause an issue with the VueJS component rendering, but I'll see if there is an alternative technique.

I'll take a look at the other items in the report and see where we can improve the HTML generation.

**Comment by throwaway-d** [comment]
By the way, I wanted to ask why my another issue about accessibility got deleted.

**Comment by geoffreymcgill** [comment]
> By the way, I wanted to ask why my another issue about accessibility got deleted.

The other post looked like a duplicate. Sorry about that. 

Here's the content if you want to recreate. 

```
Subject: PageSpeed Insights: Accessibility suggestions

https://codeberg.org/rimgo/docs/issues/7
I've put Rimgo website through a website checker called PageSpeed Insights for some performance and accessibility suggestions to improve Rimgo website. I contacted the owner of Rimgo website about those suggestions. But the owner told me that all HTML is written by Retype, the owner didn't have any control over generated HTML.

Retype generates the website, can't do anything about those suggestions. Those speed suggestions need to be added to Codeberg Pages, [Codeberg/pages-server#223](https://codeberg.org/Codeberg/pages-server/issues/223)

So I think I should report the suggestions to Retype instead, right?

Here are the accessibility suggestions: https://pagespeed.web.dev/analysis/https-rimgo-codeberg-page-docs/ojicpsz30p?form_factor=mobile
```

**Comment by throwaway-d** [comment]
> The other post looked like a duplicate. Sorry about that.
> Here's the content if you want to recreate.

No problem! Thank you for explaining and helping me to recreate! :D
Some people had closed my issues without explaining why they was closed.

**Comment by geoffreymcgill** [comment]
> Unlike some other repos which people had closed my issues without explaining why they was closed; you are really supportive and explanative!

Your two posts used near similar language and were submitted near the same time. I quickly scanned the two and assumed they were duplicated. Plus your username is `throwaway-d`, so based on that, I really didn't want to invest much time. Because of your username, I can easily see why other repo owners would close/delete your issues without explaining why.

**Comment by geoffreymcgill** [comment]
We completed some clean up of the HTML syntax. The enhancements will be included in the upcoming Retype v3.1 release.

The `v-cloak` and `v-pre` are Vue directives and removed once the Vue parsing is complete. The reports are generated from the raw HTML and not post-Vue processing. The `<template #title>` and `<template #sidebar-footer>` items are VueJS related too. 

There is still work to be done with the HTML syntax, especially related to Accessibility, but we will continue to make improvements in future releases.

Hope this helps.

**Comment by throwaway-d** [comment]
Thank you for your work and pointing out my username's problem! :)
I might change my username soon.
